src/GPhoCS.cpp

```diff
@@ -2357,13 +2120,19 @@ void setupMcRefFiles() {
   }
   if (isTauBoundsActivated()) {
     ioSetup.tauBoundsFile = fopen(ioSetup.tauBoundsFileName, "w");
-
     if (ioSetup.tauBoundsFile == NULL) {
-      fprintf(stderr, "Error: Could not open tau bounds file %s.\n",
-              ioSetup.tauBoundsFileName);
+      fprintf(stderr, "Error: Could not open tau bounds file %s.\n", ioSetup.tauBoundsFileName);
       exit(0);
     }
     printTauBoundsHeader(ioSetup.tauBoundsFile);
+
+    ioSetup.refMigStatsFile = fopen(ioSetup.refMigStatsFileName, "w");
+    if (ioSetup.refMigStatsFile == NULL) {
+      fprintf(stderr, "Error: Could not open refMigStatsFile %s.\n", ioSetup.refMigStatsFileName);
+      exit(0);
+    }
+    printRefMigStatsHeader(ioSetup.refMigStatsFile);
+
   }
```

Comment:
Eventually, I'd like all stats (comb, clade, hyp, bounds. etc.) to be printed in a single file.  G-PhoCS should produce one trace file for parameters and one for model comparison.